lms/djangoapps/hera/admin.py

```diff
@@ -13,7 +13,38 @@ class OnboardingAdmin(admin.ModelAdmin):
     """
     Reflection of the hera model on the admin panel.
     """
-    pass
+
+    def get_queryset(self, request):
+        """
+        Create a default onboarding object in case we have none yet.
+        """
+        qs = super(OnboardingAdmin, self).get_queryset(request)
+
+        if not qs:
+            default_onboarding = self.model()
+            default_onboarding.save()
+
+            qs = super(OnboardingAdmin, self).get_queryset(request)
+
+        return qs
+
+    def has_add_permission(self, request):
+        """
+        Allow to add Onboarding only if no objects exist.
+        """
+        return not self.model.objects.exists()
+
+    def get_actions(self, request):
+        """
+        Remove 'delete_selected' action from the list of available actions.
+        """
+        delete_action = 'delete_selected'
```

Comment:
maybe it worths returning an empty list, don't think we will be adding any actions here.
```suggestion
        return []
```

Reply:
Ok, will remove it.

Reply:
Not get it. But it is because I am not in the context.